boards/arm/stm32f7/nucleo-144/configs/f722-can/Make.defs

```diff
@@ -0,0 +1,39 @@
+############################################################################
+# boards/arm/stm32f7/nucleo-144/configs/f746-nsh/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(TOPDIR)/.config
```

Comment:
why not share the same Make.defs for all configs

Reply:
All configurations in stm32f7/nucleo-144 look like this. 
Personally, I think stm32f7/nucleo-144 should be divided into separate boards as it is done for the rest of Nucleo boards.
The current solution is misleading - there are nucleo-144 boards that are not based on stm32f7.  
I don't know if any other boards family has a similar organization.